cript_graph_json/JSON/cao_protein.json

```diff
@@ -1,2483 +1,5572 @@
-                "type": "null",
-                "ingredients": [
-                  {
-                    "node": "Ingredient",
-                    "material": "Soy protein isolate",
-                    "quantities": [
-                      "null"
-                    ]
-                  },
-                  {
-                    "node": "Ingredient",
-                    "material": "water",
-                    "quantities": [
-                      "null"
-                    ]
-                  }
-                ],
-                "products": [
-                  "SPI supernatant"
-                ]
-              },
-              {
-                "node": "Process",
-                "name": "PPI Centrifugation",
-                "type": "null",
-                "ingredients": [
-                  {
-                    "node": "Ingredient",
-                    "material": "Pea protein isolate",
-                    "quantities": [
-                      "null"
-                    ]
-                  },
-                  {
-                    "node": "Ingredient",
-                    "material": "water",
-                    "quantities": [
-                      "null"
-                    ]
-                  }
-                ],
-                "products": [
-                  "PPI supernatant"
-                ]
-              },
-              {
-                "node": "Process",
-                "name": "WPI Methacrylation",
-                "type": "null",
-                "prerequisite_process": [
-                  {
-                    "node": "Process",
-                    "name": "WPI Centrifugation",
-                    "type": "null",
-                    "ingredients": [
-                      {
-                        "node": "Ingredient",
-                        "material": "Whey protein isolate",
-                        "quantities": [
-                          "null"
-                        ]
+  "node": [
+    "Project"
+  ],
+  "name": "CRIPT End-to-End Test Papers",
+  "collection": [
+    {
+      "node": [
+        "Collection"
+      ],
+      "name": "Cao et al. Protein Elastomers",
+      "experiment": [
+        {
+          "node": [
+            "Experiment"
+          ],
+          "name": "Methacrylation",
+          "process": [
+            {
+              "node": [
+                "Process"
+              ],
+              "name": "WPI Centrifugation",
+              "type": "multistep",
+              "ingredient": [
+                {
+                  "node": [
+                    "Ingredient"
+                  ],
+                  "material": {
+                    "node": [
+                      "Material"
+                    ],
+                    "name": "Whey protein isolate",
+                    "uid": "_:Whey protein isolate"
+                  },
+                  "quantity": [
+                    "null"
+                  ]
```

Comment:
```suggestion
```

Reply:
I'll put the into my list to give to the backend 

```json
                  "quantity": [
                    "null"
                  ]
```

Reply:
just made a push to change it from `"null"` to `null`